tests/ghost_test.go

```diff
@@ -0,0 +1,346 @@
+package tests
+
+import (
+	"database/sql"
+	"flag"
+	"fmt"
+	"net/url"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/bytebase/bytebase/resources/mysql"
+	"github.com/github/gh-ost/go/base"
+	"github.com/github/gh-ost/go/logic"
+	ghostsql "github.com/github/gh-ost/go/sql"
+	_ "github.com/go-sql-driver/mysql"
+	"github.com/openark/golib/log"
+)
+
+// ghostRun runs gh-ost
+// ghostRun is the original gh-ost main function with
+// some simplications since we are using as a library
+func ghostRun(args []string) error {
+	os.Args = args
+	migrationContext := base.NewMigrationContext()
+	flag.StringVar(&migrationContext.InspectorConnectionConfig.Key.Hostname, "host", "127.0.0.1", "MySQL hostname (preferably a replica, not the master)")
```

Comment:
We probably just need a simple test without allowing test runners to specify these flags.

We should just set the values that we want in migrationContext.